cosmos/node/config.go

```diff
@@ -44,133 +39,20 @@ func recursiveModifyToml(c map[string]any, modifications Toml) error {
 	}
 	return nil
 }
-func GenerateDefaultClientConfig(chainID string) Toml {
```

Comment:
As discussed, let's keep these and overwrites/add to the configs with the provided input